skrf/vectorFitting.py

```diff
@@ -1980,7 +1982,7 @@ def get_new_subckt_identifier():
             f.write('L1 n_pos 1 {ind}\n')
             f.write('C1 1 2 {cap}\n')
             f.write('R1 2 n_neg {res}\n')
-            f.write('G1 n_pos n_neg 1 2 {gm} m={mult}\n')
+            f.write('G1 n_pos n_neg 1 2 {gm * mult}\n')
```

Comment:
Good idea!
It seems the `mult` parameter only exists in ngspice. But does it always work to have a negative transductance (in other versions than LTspice)? Based on the docs, I'm not entirely sure. Did you test this? Otherwise, we can also flip the controlling nodes in case of `multi = -1`.

Berkeley SPICE: http://bwrcs.eecs.berkeley.edu/Classes/IcBook/SPICE/UserGuide/elements.html#462
ngspice (chapter 4.2.1): https://ngspice.sourceforge.io/docs/ngspice-37-manual.pdf

Reply:
I tested with Qucs (ngspice backend) and this worked. However this was my first try with qucs, maybe you could double check. :)

Reply:
Ok. Sorry for the delay. I'm running some tests now.

Reply:
Works well in both LTSpice and ngspice